scripts/japicmp.sh

```diff
@@ -20,8 +20,8 @@ SCRIPT=$(basename "${BASH_SOURCE:-stidn}")
 if [ $# -lt 1 ] || [ $# -gt 3 ]; then
   echo "# This script compares versions for binary backward compatibility."
   echo "# Usage: ${SCRIPT} <old_version> (<new_version> (<group_id>))"
-  echo "# if <new_version> unspecified then comparison will be made againt local build"
-  echo "# if <group_id> unspecified then local dir gradle 'group' property will be used"
+  echo "# if optional <new_version> unspecified then comparison will be made againt local build"
+  echo "# if optional <group_id> unspecified then local dir gradle 'group' property will be used"
```

Comment:
It might be useful to be able to run a local build with `group_id`. Consider using "local" as a keyword for the local build to allow using 3rd parameter. We can also clarify in this message that if "local" is used, users should run `./gradlew build` before running this script.